executor/show.go

```diff
@@ -628,6 +636,15 @@ func (e *ShowExec) fetchShowCreateTable() error {
 					buf.WriteString(" DEFAULT CURRENT_TIMESTAMP")
 				default:
 					defaultValStr := fmt.Sprintf("%v", defaultValue)
+					// If column is timestamp, and default value is not current_timestamp, should convert the default value to the current session time zone.
+					if col.Tp == mysql.TypeTimestamp && defaultValStr != types.ZeroDatetimeStr {
+						timeValue, err := table.GetColDefaultValue(e.ctx, col.ToInfo())
```

Comment:
What's the difference between col.GetDefaultValue and table.GetColDefaultValue?

Reply:
for timestamp column, col.GetDefaultValue will return the default `string` value. `table.GetColDefaultValue` will parse the `col.GetDefaultValue` string to time, and then convert to the session time zone.